js/siteorigin-panels/dialog/row.js

```diff
@@ -174,8 +175,10 @@ module.exports = panels.view.dialog.extend( {
 				handle
 					.appendTo( newCell )
 					.dblclick( function () {
-						var t = thisDialog.row.cells[i] + thisDialog.row.cells[i - 1];
-						thisDialog.row.cells[i] = thisDialog.row.cells[i - 1] = t / 2;
+                        var prevCellModel = thisDialog.row.cells.at(i - 1);
```

Comment:
More space indents messing with our code style. I never see these in the editor, but on Github they make the code look strange. There are a few others in this file. Could just do a Code Reformat to fix all of them.